### PR TITLE
Improve code about user avatar

### LINKED ISSUE #207: Fetch user avatars images from rust.
We would need to fetch user avatars from backend to render on flutter chat ui via the ```RoomMessage``` API.

### PR DESCRIPTION
**Pull request by bitfriend**
This PR fixes #207.

`UserId` can be accessible on rust side, not flutter side.
So I replaced `UserId` with `String` in parameter of `room.get_member()`, because `String` is accessible from flutter side.
And implemented sample code about chat avatar.

### CODE REVIEW

File: app/lib/screens/HomeScreens/ChatScreen.dart [hunk 1]
```diff
@@ -118,6 +119,29 @@ class _ChatScreenState extends State<ChatScreen> {
     );
   }
 
+  Future<FfiBufferUint8> _userAvatar(String userId) async {
+    final member = await widget.room.getMember(userId);
+    return member.avatar();
+  }
+
+  Widget _avatarBuilder(String userId) {
```
Comment (gtalha07) [inline app/lib/screens/HomeScreens/ChatScreen.dart:127]: I think I have removed this as we don't need this custom method to apply avatars. The package already supports it and also have algorithm to change user avatar background colors based on ```userID```.  Or this is something we needed here?

File: app/lib/screens/HomeScreens/ChatScreen.dart [hunk 2]
```diff
@@ -118,6 +119,29 @@ class _ChatScreenState extends State<ChatScreen> {
...
+    return GetBuilder<ChatController>(
+      id: 'Avatar',
+      builder: (ChatController controller) {
+        return Padding(
+          padding: const EdgeInsets.only(right: 10),
+          child: CustomAvatar(
+            avatar: _userAvatar(userId),
+            displayName: null,
+            radius: 15,
```
Comment (gtalha07) [inline app/lib/screens/HomeScreens/ChatScreen.dart:136]: Also the image avatar radius appear smaller and text avatar appears bigger. Is there a way to make it even to image avatar size?

File: app/lib/screens/HomeScreens/ChatScreen.dart (hunk 1 above)
Comment (bitfriend) [reply app/lib/screens/HomeScreens/ChatScreen.dart:127]: This function is needed to get avatar from user id.

File: app/lib/screens/HomeScreens/ChatScreen.dart (hunk 2 above)
Comment (bitfriend) [reply app/lib/screens/HomeScreens/ChatScreen.dart:136]: What if this code should work on desktop or web, not just mobile?

File: app/lib/screens/HomeScreens/ChatScreen.dart (hunk 1 above)
Comment (gtalha07) [reply app/lib/screens/HomeScreens/ChatScreen.dart:127]: Then its okay to use. Another problem with this is that text avatars appear bigger than usual.

File: app/lib/screens/HomeScreens/ChatScreen.dart (hunk 2 above)
Comment (gtalha07) [reply app/lib/screens/HomeScreens/ChatScreen.dart:136]: On mobile, it looks weirdly. Like images avatars appear smaller and text one bigger. Not a good looking conversation UI. :)

Comment (gtalha07) [reply app/lib/screens/HomeScreens/ChatScreen.dart:136]: Anyways we should do something about text avatars size. They should be even depending upon platforms.

Comment (bitfriend) [reply app/lib/screens/HomeScreens/ChatScreen.dart:136]: Is it impossible to resize image on flutter side?

**Review by gnunicorn — APPROVED** [review]

File: app/lib/screens/HomeScreens/ChatScreen.dart (hunk 1 above)
Comment (bitfriend) [reply app/lib/screens/HomeScreens/ChatScreen.dart:127]: Of course, I can add `width` & `height` parameters to `Member::avatar()` function.
But I think it is efficient to resize with Image class method on flutter.
Is it impossible to resize with Image class method on flutter?
We have to think about web or desktop too.

**Review by gtalha07 — APPROVED** [review]

**Pull request merged**